src/views/Home.vue

```diff
@@ -13,7 +13,22 @@
       <div class="rf-col-8">
 
         <IntroSection></IntroSection>
-        <SearchBar></SearchBar>
+
+        <SearchBar>
+             <form>
```

Comment:
Suggestion : normalement, se brancher sur l'événement `submit` devrait suffire, il devrait donc possible de supprimer les branchements sur les événements `keydown` et `click`.

```suggestion
             <form @submit.stop.prevent="goToResearch()">
```